pyproject.toml

```diff
@@ -20,6 +20,7 @@ dependencies = [
     "simple-parsing>=0.1.1",
     "flatten-dict>=0.4.1",
     "wandb>=0.15.0",
+    "python-dotenv",
```

Comment:
@levmckinney Do we want to add this as a dependency? I don't have a strong opinion on the matter but I've never used this package.

Other than that LGTM